tests/lora/test_lora_layers_peft.py

```diff
@@ -1205,6 +1205,13 @@ class StableDiffusionLoRATests(PeftLoraLoaderMixinTests, unittest.TestCase):
         "latent_channels": 4,
     }
 
+    def tearDown(self):
+        import gc
```

Comment:
I think it's fine to import gc at the top since it comes packaged in the standard Python installation.